content/en/integrations/guide/azure-manual-setup.md

```diff
@@ -0,0 +1,234 @@
+
+To assign access at the Management Group level, navigate to **Management Groups** and select the Management Group that contains the set of subscriptions you would like to monitor.  
+**Note**: Assigning access at the Management Group level means that any new subscriptions added to the group are automatically discovered and monitored by Datadog.
+
+{{< img src="integrations/azure/azure_management_groups_icon.png" alt="Management groups icon" popup="true" style="width:25%">}}
+
+To configure monitoring for the entire tenant, assign access to the **Tenant Root Group**.
+
+2. Click on the subscription you would like to monitor.
+3. Select **Access control (IAM)** in the subscription menu and click **Add** > **Add role assignment**:
+
+    {{< img src="integrations/azure/azure-add-role.png" alt="Add Role Assignment" popup="true" style="width:80%">}}
+
+4. For **Role**, select **Monitoring Reader**. Under **Select**, choose the name of the Application you just created:
+
+5. Click **Save**.
+6. Repeat this process for any additional subscriptions you want to monitor with Datadog.
+**Note**: Users of Azure Lighthouse can add subscriptions from customer tenants.
+
+**Note**: Diagnostics must be enabled for ARM deployed VMs to collect metrics, see [Enable diagnostics][11].
+
+#### Completing the integration
+
+1. Under **App Registrations**, select the App you created, copy the **Application ID** and **Tenant ID**, and paste the values in the [Datadog Azure integration tile][10] under **Client ID** and **Tenant ID**.
+2. For the same app, go to **Manage** > **Certificates and secrets**.
+3. Add a new **Client Secret** called `datadogClientSecret`, select a timeframe for **Expires**, and click **Add**:
+
+    {{< img src="integrations/azure/Azure_client_secret.png" alt="Azure client secret" popup="true" style="width:80%">}}
+
+4. When the key value is shown, copy and paste the value in the [Datadog Azure integration tile][10] under **Client Secret** and click **Install Integration** or **Update Configuration**.
+
+**Note**: Your updates to the Azure configuration can take up to 20 minutes to be reflected in Datadog.
+
+### Configuration
+
+To limit resource collection for Azure-based hosts, open the integration tile for Azure. Select the **Configuration** tab, then open **App Registrations**. Enter a list of tags in the text box under **Metric Collection Filters**.
+
+This list of tags in `<KEY>:<VALUE>` form is separated by commas and defines a filter used while collecting metrics. Wildcards such as `?` (for single characters) and `*` (for multiple characters) can also be used. 
+
+Only VMs that match one of the defined tags are imported into Datadog. The rest are ignored. VMs matching a given tag can also be excluded by adding `!` before the tag. For example:
+
+```text
+datadog:monitored,env:production,!env:staging,instance-type:c1.*
+```
+
+### Monitor the integration status
```

Comment:
This applies to the app registration type, both through the manaul set up & programmatic - it looks like it's missing in programmatic

Reply:
Thanks for pointing that out -- I added the monitoring section and changed the logs section to link to the new logs guide.